Ports/c-ares/package.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='c-ares'
+version='1.19.0'
+files="https://c-ares.org/download/c-ares-${version}.tar.gz c-ares-${version}.tar.gz
+https://c-ares.org/download/c-ares-${version}.tar.gz.asc c-ares-${version}.tar.gz.asc"
+auth_type='sig'
+auth_import_key="27EDEAF22F3ABCEB50DB9A125CC908FDB71E12C2"
+auth_opts=("c-ares-${version}.tar.gz.asc" "c-ares-${version}.tar.gz")
+useconfigure='true'
+configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt")
+
+configure() {
+	mkdir -p c-ares-build
+	cmake -G Ninja \
+	"${configopts[@]}" \
+	-S "c-ares-${version}" \
+	-B c-ares-build
+}
+
+build() {
+	ninja -C c-ares-build -j`nproc`
```

Comment:
We should avoid using `nproc` directly, some users may want to use a smaller number of build jobs. We already define a `MAKEJOBS` variable, which one should use for ports scripts instead.